### PR TITLE
[percep] Added ZED and PCL testing capabilities to Travis

### PR DESCRIPTION
**Pull request by mmartin4972**
Reconfigured Travis to download pre-built image from DockerHub and run code in image. Added CUDA, ZED, and PCL installation instructions to Dockerfile, to create image capable of compiling new version of ZED SDK and PCL. Image must be pre-built and pushed to DockerHub to be run.

### CODE REVIEW

**Review by raytitan — COMMENTED** [review]
How much longer does integration testing take now?

**Comment by mmartin4972** [comment]
You mean like how long does it take to run Travis?

**Comment by mmartin4972** [comment]
This way takes 20min. https://travis-ci.com/github/umrover/perception-travis-test-repo/builds/213085530

**Review by DavidSmith166 — APPROVED** [review]

**Comment by mmartin4972** [comment]
I did want to note that my one concern with downloading the Docker Image for each Travis test is that since it's a 15Gb file we can easily exceed daily bandwidth limitations. However, based on their website DockerHub allows 100 image pulls per 6 hours, which I don't believe we will exceed. I just wanted to let it be known that if something were to go wrong it would most likely be that we exceeded the download limit

**Review by cgiger00 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -55,29 +90,67 @@ RUN cd /opt && \
...
 
-ENV ANSIBLE_GATHERING smart
-ENV ANSIBLE_HOST_KEY_CHECKING false
-ENV ANSIBLE_RETRY_FILES_ENABLED false
-ENV ANSIBLE_ROLES_PATH /ansible/playbooks/roles
-ENV ANSIBLE_SSH_PIPELINING True
-ENV PYTHONPATH /ansible/lib
-ENV PATH /ansible/bin:$PATH
-ENV ANSIBLE_LIBRARY /ansible/library
+#Install VTK
```
Comment (raytitan) [inline Dockerfile:116]: what's vtk?

**Review by raytitan — COMMENTED** [review]
One day I do hope to move 3rd party dependency installations to jarvis ([for example this](https://github.com/raytitan/mrover-workspace/tree/eigen))

File: Dockerfile (hunk 1 above)
Comment (mmartin4972) [reply Dockerfile:116]: It's a PCL dependency

Comment (mmartin4972) [reply Dockerfile:116]: It's a dependency. That's a good goal

**Comment by mmartin4972** [comment]
When are those jarvis dependencies built? Is the idea that jarvis would install PCL?

**Review by raytitan — APPROVED** [review]

**Review by DavidSmith166 — APPROVED** [review]

**Review by cgiger00 — APPROVED** [review]

**Pull request merged**